docs/tutorial.md

```diff
@@ -154,3 +156,52 @@ mothulity ~/MiSeq_SOP -r bash -n my_first_mothulity_project
 ```-n my_first_mothulity_project``` is used to name files, directories and give a title the final output.
 
 The output is placed in ```~/MiSeq_SOP/analysis/OTU/analysis_my_first_mothulity_project.html``` and should look like [this](./analysis-example/analysis-my-first-mothulity-project.html)
+
+
+## [Slurm Workload Manager](https://www.schedmd.com)
+
+
+```mothulity``` can be conveniently used with [Slurm Workload Manager](https://www.schedmd.com) so it is good idea to use on your HPC/computing facility. It requires two steps:
```

Comment:
I believe it should be rather:
(...) so it is a good idea to use **it** on(...) OR "which makes it easy to apply to".

Reply:
Very perceptive! Fixed this typo! :+1: